_includes/benefits.html

```diff
@@ -10,7 +10,7 @@
             <div class="benefit-image"><img alt="{{ benefit.title }} logo" src="{{ benefit.image }}"/></div>
             {% endif %}
             <h2 class="benefit-title">{{ benefit.title }}</h2>
-            <div class="benefit-content">{{ benefit.description }}</div>
+            <div class="benefit-content text-justified">{{ benefit.description }}</div>

```

Comment:
The text on each card is aligned to the bottom, I think it's a bit out of a place. Maybe we should align it to the top or middle.

<details>
<summary>Bottom aligned (margin-bottom: 0; margin-top: auto)</summary>

![image](https://user-images.githubusercontent.com/20901158/98470408-07b27100-21ee-11eb-82c4-00ab7567f2bd.png)

</details>

<details>
<summary>Top aligned (margin-bottom: auto; margin-top: 0)</summary>

![image](https://user-images.githubusercontent.com/20901158/98470416-1862e700-21ee-11eb-9255-64025ff3a186.png)

</details>

<details>
<summary>Middle aligned (margin-bottom: auto; margin-top: auto)</summary>

![image](https://user-images.githubusercontent.com/20901158/98470439-47795880-21ee-11eb-828f-8399f874c2a9.png)

</details>